web/modules/custom/rsvplist/src/Form/RSVPForm.php

```diff
@@ -21,32 +22,25 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
-    /**
-     * Gives the fiully Loaded Node Object of the current node
-     */
+    
+    // Gives the fiully Loaded Node Object of the current node.
     $node = \Drupal::routeMatch()->getParameter('node');
 
```

Comment:
I dont know how this code passed Drupal codesniffer, since this way of writing code in classes is not the standard.
Please check again how you are running the command for phpcs.

Reply:
I fixed it